Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/bugreporting/DebugLogFragmentTest.kt

```diff
@@ -0,0 +1,134 @@
+package de.rki.coronawarnapp.bugreporting
+
+import android.content.ContentResolver
+import androidx.lifecycle.MutableLiveData
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import dagger.Module
+import dagger.android.ContributesAndroidInjector
+import de.rki.coronawarnapp.bugreporting.debuglog.DebugLogger
+import de.rki.coronawarnapp.bugreporting.debuglog.export.SAFLogExport
+import de.rki.coronawarnapp.bugreporting.debuglog.internal.LogSnapshotter
+import de.rki.coronawarnapp.bugreporting.debuglog.ui.DebugLogFragment
+import de.rki.coronawarnapp.bugreporting.debuglog.ui.DebugLogViewModel
+import de.rki.coronawarnapp.nearby.ENFClient
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.spyk
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import testhelpers.BaseUITest
+import testhelpers.Screenshot
+import testhelpers.SystemUIDemoModeRule
+import testhelpers.TestDispatcherProvider
+import testhelpers.launchFragment2
+import testhelpers.launchFragmentInContainer2
+import testhelpers.takeScreenshot
+import tools.fastlane.screengrab.locale.LocaleTestRule
+
+@RunWith(AndroidJUnit4::class)
+class DebugLogFragmentTest : BaseUITest() {
+
+    @MockK lateinit var debugLogger: DebugLogger
+    @MockK lateinit var enfClient: ENFClient
+    @MockK lateinit var bugReportingSettings: BugReportingSettings
+    @MockK lateinit var logSnapshotter: LogSnapshotter
+    @MockK lateinit var safLogExport: SAFLogExport
+    @MockK lateinit var contentResolver: ContentResolver
+
+    private lateinit var inactiveViewModel: DebugLogViewModel
+    private lateinit var activeViewModel: DebugLogViewModel
+
+    @Rule
+    @JvmField
+    val localeTestRule = LocaleTestRule()
+
+    @get:Rule
+    val systemUIDemoModeRule = SystemUIDemoModeRule()
+
+    @Before
+    fun setup() {
+
```

Comment:
Why do you add an empty line at the start of functions?

Reply:
Good point, will be removed ;)